package.json

```diff
@@ -92,12 +92,13 @@
     "jsinspect": "^0.12.6",
     "minimatch": "^3.0.4",
     "mock-stdin": "^0.3.0",
-    "prettier": "^1.5.2",
+    "prettier": "1.5.2",
     "temp": "^0.8.3",
     "webpack": "^2.1.0-beta.25",
     "yargs": "^6.3.0"
   },
   "resolutions": {
+    "chalk@2": "2.3.2",
```

Comment:
Rather than doing this, can you update the lockfile to merge the three `chalk` entries together, since they all seem to accept `2.3.2` as valid resolution (I know there's a "do no edit" warning at the top of the file, but it's actually safe to do if you know what you're doing)?